app/views/mappings/_show_line.html.haml

```diff
@@ -1,5 +1,6 @@
 - process = map.process || {}
-- source = "#{map.source} #{process[:source_name]}"
+- type_tooltip = "#{map.source} #{process[:source_name]}".strip
```

Comment:
Here I flipped source and type cause, the source here is the type and the type is the source.
source like: 'internal/external'
type like: 'loom, skos..'